.changeset/lovely-chefs-remain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@talend/design-docs': minor
+'@talend/react-stepper': minor
+'@talend/react-forms': minor
```

Comment:
Looks not clear for me if it's a breaking change or not.
(i think it is if we load RHF as a UMD in the apps)